geniza/corpus/locale/ar/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,41 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR The Trustees of Princeton University
+# This file is distributed under the same license as the geniza package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
We don't appear to be able to change the wonderful `SOME DESCRIPTIVE TITLE` or the authors list. I think the authors might be something translators are expected to edit as they contribute to the file, since it's a record of who helped localize something. Theoretically tools that help with this process like [poedit](https://poedit.net/) would handle this part.